python/lsst/afw/math/detail/detailLib.py

```diff
@@ -1 +1,3 @@
-from __future__ import absolute_import
\ No newline at end of file
+from __future__ import absolute_import
```

Comment:
I'm not sure what this comment is about. This is the standard line we have been using for importing in every *Lib.py file.